Packs/qualys/Integrations/Qualysv2/Qualysv2.py

```diff
@@ -1413,6 +1713,47 @@ def build_multiple_values_parsed_output(**kwargs) -> Tuple[List[Any], str]:
     return parsed_output, readable_output
 
 
+@logger
+def build_host_list_detection_outputs(kwargs) -> Tuple[List[Any], str]:
```

Comment:
It seems like there are parts of this functions that are copied from `build_multiple_values_parsed_output`.
Like lines 1726-1731, 1736-1738, 1740-1741. 
Please make this functionalities separate functions, where possible, and use the functions in both `build_host_list_detection_outputs` and  `build_multiple_values_parsed_output`.

Reply:
No need to separate the functions here. It is an overreach and will be hard to read and debug later.